portal-kernel/src/com/liferay/portal/kernel/service/ServiceContextFactory.java

```diff
@@ -150,13 +149,6 @@ public static ServiceContext getInstance(
 		return serviceContext;
 	}
 
-	public static ServiceContext getInstance(
```

Comment:
directly remove this method, because uploadPortletRequest is the child class of HttpServletRequest, and we can use getInstance(String, HttpServletRequest) directly